src/Services/SearchService.php

```diff
@@ -465,7 +483,7 @@ private function getVariationIdForRedirect(string $query, array $documents)
         foreach ($documents as $document) {
             $variation = $document['data']['variation'];
             $barcodes = $document['data']['barcodes'] ?? [];
-            $variationPrice = $this->getCheapestPrice($document['data']['salesPrices']);
+            $variationPrice = $document['data']['prices']['default']['price']['value'] ?: 0;
```

Comment:
Is this actually the cheapest price?

Reply:
Note: Also check why unit test isn't failing.

Reply:
> Is this actually the cheapest price?

Yes, the cheapeast price gets set on line 518, so the logic works as before, just without need to parse the `salesPrices` array.